packages/devtools_app/lib/src/screens/logging/logging_screen_v2/logging_controller_v2.dart

```diff
@@ -634,15 +627,21 @@ class LogDataV2 with SearchableDataMixin {
 
   String? get details => _details;
 
-  bool get needsComputing => detailsComputer != null;
+  ValueListenable<bool> get detailsComputed => _detailsComputed;
+  final ValueNotifier<bool> _detailsComputed = ValueNotifier<bool>(false);
 
   Future<void> compute() async {
-    _details = await detailsComputer!();
-    detailsComputer = null;
+    if (!detailsComputed.value) {
+      if (detailsComputer != null) {
+        _details = await detailsComputer!();
+      }
+      detailsComputer = null;
+      _detailsComputed.value = true;
```

Comment:
do we still want to set this to true even if `detailsComputer()` was never called?

Reply:
if we got to this point then any work that needed to be done is done, so I think this is fine :)